entity-framework/core/miscellaneous/async.md

```diff
@@ -18,6 +18,9 @@ For more information, see [the general C# asynchronous programming docs](/dotnet
 > [!WARNING]
 > EF Core doesn't support multiple parallel operations being run on the same context instance. You should always wait for an operation to complete before beginning the next operation. This is typically done by using the `await` keyword on each async operation.
 
+> [!WARNING]
+> The async implementation of [Microsoft.Data.SqlClient](https://github.com/dotnet/SqlClient) unfortunately has some known issues with async on .NET Core (e.g. [#593](https://github.com/dotnet/SqlClient/issues/593), [#601](https://github.com/dotnet/SqlClient/issues/601), and others).
```

Comment:
Could "on .NET Core" be removed

Reply:
Yeah, missed that it was an issue in netfx too, will do.